ios/FluentUI.Demo/FluentUI.Demo/Demos/PopupMenuDemoController_ObjC.m

```diff
@@ -0,0 +1,174 @@
+//
+//  Copyright (c) Microsoft Corporation. All rights reserved.
+//  Licensed under the MIT License.
+//
+
+#import "PopupMenuDemoController_ObjC.h"
+#import <FluentUI/FluentUI-Swift.h>
+
+@implementation PopupMenuDemoController_ObjC
+
+- (instancetype)init {
+    self = [super init];
+    if (self != nil)
+    {
+        _selectedCityIndex = [[NSIndexPath alloc] initWithIndex:0];
+    }
+    return self;
+}
+
+- (void)viewDidLoad {
```

Comment:
because this viewcontroller isn't democontroller, most of the logic of view setup should be loadView()